dashboard/src-tauri/src/main.rs

```diff
@@ -106,17 +128,22 @@ async fn main() {
                     if let Some(tx) = shutdown_tx.lock().unwrap().take() {
                         tx.send(()).unwrap();
                     }
```

Comment:
Why are we removing this app.exit(0)?

Reply:
i think i accidentally deleted that in a merge, added it back now